### PR TITLE
NO-JIRA: Update owners file after I changed my GitHub username

### PR DESCRIPTION
**Pull request by christoph-jerolimov**
/assign @spadgett @jhadvig 
/cc @vikram-raj @lokanandaprabhu @Lucifergene @invincibleJai

### CODE REVIEW

File: frontend/packages/dev-console/src/components/import/devfile/__tests__/devfile-utils.spec.ts [hunk 1]
```diff
@@ -326,7 +326,7 @@ describe('devfile-utils', () => {
     it('should use coFetch for external URIs', async () => {
       const resourceContent = await getResourceContent(
         'outerloop-deploy',
-        'https://raw.githubusercontent.com/jerolimov/devfile-sample-go-basic-remote-k8s-uri/main/outerloop-deploy.yaml',
+        'https://raw.githubusercontent.com/christoph-jerolimov/devfile-sample-go-basic-remote-k8s-uri/main/outerloop-deploy.yaml',
```
Comment (vikram-raj) [inline frontend/packages/dev-console/src/components/import/devfile/__tests__/devfile-utils.spec.ts:329]: I am getting 404 for this URL. and old URL https://raw.githubusercontent.com/jerolimov/devfile-sample-go-basic-remote-k8s-uri/main/outerloop-deploy.yaml is working fine.

File: frontend/packages/git-service/src/services/__tests__/bitbucket-service.spec.ts [hunk 2]
```diff
@@ -139,7 +139,7 @@ describe('Bitbucket Service', () => {
 
   it('should detect Dockerfile', () => {
     const gitSource: GitSource = {
-      url: 'https://bitbucket.org/jerolimov/nodeinfo',
+      url: 'https://bitbucket.org/christoph-jerolimov/nodeinfo',
```
Comment (vikram-raj) [inline frontend/packages/git-service/src/services/__tests__/bitbucket-service.spec.ts:142]: Same here getting 404 for this. But old one is working fine https://bitbucket.org/jerolimov/nodeinfo

File: frontend/packages/git-service/src/services/__tests__/bitbucket-service.spec.ts [hunk 3]
```diff
@@ -155,7 +155,7 @@ describe('Bitbucket Service', () => {
 
   it('should return exposed container port from dockerfile', () => {
     const gitSource: GitSource = {
-      url: 'https://bitbucket.org/jerolimov/nodeinfo',
+      url: 'https://bitbucket.org/christoph-jerolimov/nodeinfo',
```
Comment (vikram-raj) [inline frontend/packages/git-service/src/services/__tests__/bitbucket-service.spec.ts:158]: Old one works fine https://bitbucket.org/jerolimov/nodeinfo this one giving 404

File: frontend/packages/dev-console/src/components/import/devfile/__tests__/devfile-utils.spec.ts (hunk 1 above)
Comment (christoph-jerolimov) [reply frontend/packages/dev-console/src/components/import/devfile/__tests__/devfile-utils.spec.ts]: Reverted this change

File: frontend/packages/git-service/src/services/__tests__/bitbucket-service.spec.ts (hunk 3 above)
Comment (christoph-jerolimov) [reply frontend/packages/git-service/src/services/__tests__/bitbucket-service.spec.ts]: Reverted this change

File: frontend/packages/git-service/src/services/__tests__/bitbucket-service.spec.ts (hunk 2 above)
Comment (christoph-jerolimov) [reply frontend/packages/git-service/src/services/__tests__/bitbucket-service.spec.ts]: Reverted this change

**Comment by vikram-raj** [comment]
@christoph-jerolimov Looks like you are not in openshift org after updating the username.

**Comment by christoph-jerolimov** [comment]
> @christoph-jerolimov Looks like you are not in openshift org after updating the username.

Yep, the bot removed me, and I must request access again. Now I'm part of the org again. :grin: 

I will rebase the PR to trigger a new check.

**Comment by christoph-jerolimov** [comment]
chore
/label px-approved
/label qe-approved
/label docs-approved

**Review by vikram-raj — COMMENTED** [review]
/lgtm
/approve

**Comment by vikram-raj** [comment]
Looks like you missed to update this https://github.com/openshift/console/blob/master/OWNERS

**Pull request merged**